docs/guides/admin/docs/registration.md

```diff
@@ -0,0 +1,55 @@
+Registering your Opencast installation
+======================================
+
+Why we want you to register
+---------------------------
+
+Measuring the health, and uptake of an Open Source project can be difficult at the best of times, especially when you
+consider the privacy needs of your adopters. Registering your Opencast installation is entirely optional, but it is the
+only way to appear on [the community map](https://map.opencast.org).  It is also the easiest, and best way to provide
+Opencast with up-to-date data about who your organiation is, and how they are using Opencast.  This information enables
+us to better focus our resources, and lets us track broad-stroke usage patterns so we can make Opencast better.
+
+What we gather
+--------------
+
+We gather two different classes of information: Biographical, and Statistical.  To register, you only need to provide
+biographical data, however statistical data is strongly recommended as well.  All of the fields we gather are optional,
+so providing just your institutional name and email is definitely supported, but we plot the adopter pins on the map
+based on your city, so please ensure that is filled out!  No other biographical data is released, and we are GDPR
+compliant if you ever change your mind and wish to be removed from our records.
+
+The statistical data we gather includes:
+
+* The number of recordings, and series
+* The number of capture agents
+* The number of users
+* The number of Opencast jobs
+* The sum of the duration of your recordings
+* The number of tenants
+* The number of Opencast nodes, including their configurations
```

Comment:
How is "including their configuration" defined? We do not upload the config files, right? In any case, this should be specified.

Reply:
Good point!  I meant hardware configs (ram, cores, etc), but it's very unclear here.